labelbox/data/serialization/labelbox_v1/objects.py

```diff
@@ -23,7 +23,7 @@ class LBV1ObjectBase(LBV1Feature):
     classifications: List[Union[LBV1Text, LBV1Radio, LBV1Dropdown,
                                 LBV1Checklist]] = []
 
-    def dict(self, *args, **kwargs):
+    def dict(self, *args, **kwargs) -> Dict[str, str]:
```

Comment:
this is `Dict[str, Any]` since all of these objects will be nested json objects.